evap/staff/templates/staff_user_form.html

```diff
@@ -21,7 +21,21 @@ <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create u
             <div class="ms-auto d-print-none">
                 {% if has_due_evaluations %}
                     <div>
-                        <button type="button" class="btn btn-sm btn-light" onclick="resendEmailModalShow({{ form.instance.id }}, '{{ form.instance.full_name|escapejs }}');">{% trans 'Resend evaluation started email' %}</button>
+                        <form reload-on-success method="POST" action="{% url 'staff:user_resend_email' %}">
+                            {% csrf_token %}
+
+                            <confirmation-modal type="submit" name="user_id" value="{{ form.instance.id }}" confirm-button-class="btn-success">
```

Comment:
```suggestion
                            <confirmation-modal type="submit" name="user_id" value="{{ form.instance.id }}" confirm-button-class="btn-primary">
```
Don't know why it was `success` though